.env.template

```diff
@@ -1 +1,2 @@
+PUBLIC_URL=.
```

Comment:
Using the public folder seems to be a discouraged practice too https://create-react-app.dev/docs/using-the-public-folder/  as it introduces untracked assets.

> Note that we normally encourage you to import assets in JavaScript files instead. For example, see the sections on [adding a stylesheet](https://create-react-app.dev/docs/adding-a-stylesheet) and [adding images and fonts](https://create-react-app.dev/docs/adding-images-fonts-and-files). This mechanism provides a number of benefits:
> 
> Scripts and stylesheets get minified and bundled together to avoid extra network requests.
> Missing files cause compilation errors instead of 404 errors for your users.
> Result filenames include content hashes so you don’t need to worry about browsers caching their old versions.